docs/Administration.rst

```diff
@@ -4621,6 +4621,21 @@ Then select the page *News* > *Details* from the selector "Show in page"
 
 Finally tick the checkbox "Restricted by user role" and click on "Save" button. 
 
+.. note:: An issue was detected in the 5.3 version for the display of pinned news for users who are not members of the post space. This issue is due to incompatibility between News and Web-pack Addons. 
+To work around the problem you can either uninstall Web-pack addon as explained :ref:`here <eXoAddonsGuide.WCM.Installation.uninstall>` or replace the exo:news view template as explained in the following section.
+
+Go to 'Administration > Content > Content Administration' then 'Templates > Documents'. 
+
+|image354| 
+
+From the Documents interface, click on 'edit' action for News template
+
+|image355| 
+
+A popup "View & Edit Template" is displayed. Go to 'View' tab and copy paste the template from the following :ref:`link <https://raw.githubusercontent.com/exoplatform/news/1.1.1/webapp/src/main/webapp/WEB-INF/conf/news/nodetypes-templates/news/views/view1.gtmpl>`__ in the content field
```

Comment:
I think there is a missing step here, after going to the View tab, the user must click on the edit button of the view template.

Reply:
Updated